rfcs/20211207-softmax-v2/README.md

```diff
@@ -0,0 +1,144 @@
+# RFC: Softmax v2
+
+## Introduction
+
+Several primitives in the library were designed with operation descriptors containing only `data_desc` member, which covers source and destination memory descriptors.
```

Comment:
A quibble, but should this be 80 characters wide?

Reply:
I found it faster to edit a second and third reading and insert your nice suggestions, too.
I've decided I'll do them 80 symbols long after your review. Thanks.